server.js

```diff
@@ -0,0 +1,18 @@
+var path = require('path');
```

Comment:
Reason for `var` vs. `const` here? DevDeps require node LTS 4+ regardless, so ES6 will run ok 😄